src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/SearchUsersIntegrationTest.java

```diff
@@ -135,7 +135,8 @@ void shouldReturn400WhenSearchStringDoesNotContainRequiredLength(String role) {
                 userSearchRequest, role, false);
         assertThat(response).containsEntry("http_status", "400");
         var responseBody = (String) response.get("response_body");
-        assertTrue(responseBody.contains("should have atleast 3 characters"));
+        assertTrue(responseBody.contains("searchString should contains atleast 3 with letters, Apostrophe and Hyphen "
+                + "only and white space allowed"));
```

Comment:
The error message wording is a bit confusing - can we replace this error message with `"searchString must be at least 3 characters and can only contain letters, apostrophe, hypen and white space"`

Reply:
updated the error message